index.js

```diff
@@ -32,8 +32,7 @@ module.exports = ({consumerKey, consumerSecret, accessToken, accessTokenSecret,}
         const t = new Twit({
             consumer_key: consumerKey,
             consumer_secret: consumerSecret,
-            access_token: accessToken,
-            access_token_secret: accessTokenSecret,
+            app_only_auth: true
```

Comment:
I'll need to test this out first. I know that user and app contexts have different rate limits. Need to be sure there aren't any side effects for auth.